UefiTestingPkg/FunctionalSystemTests/MemoryProtectionTest/App/MemoryProtectionTestApp.c

```diff
@@ -852,330 +941,666 @@ SmmNullPointerPreReq (
 /// ================================================================================================
 /// ================================================================================================
 
+/**
+  This test case checks that page guards are present for the target memory type within Context.
+
+  The test can be run in 3 ways:
+  1. Using the Memory Attribute Protocol: This version of the test will allocate a page of the target
+     memory type and check that the page preceding and succeeding the allocated page have the EFI_MEMORY_RP

```

Comment:
Ard said that Arm's CpuDxe doesn't have a way to set `EFI_MEMORY_RP`, is this something we are able to query on the ARM64 side?